samza-core/src/main/java/org/apache/samza/clustermanager/ContainerManager.java

```diff
@@ -435,32 +459,59 @@ private boolean deQueueAction(ContainerPlacementRequestMessage requestMessage) {
 
   /**
    * A valid container placement action needs a valid processor id. Duplicate actions are handled by de-duping on uuid.
+   * If standby containers are enabled destination host requested must meet standby constraints
    *
    * @param requestMessage container placement request message
    * @return Pair<ContainerPlacementMessage.StatusCode, String> which is status code & response suggesting if the request is valid
    */
   private Pair<ContainerPlacementMessage.StatusCode, String> validatePlacementAction(ContainerPlacementRequestMessage requestMessage) {
-    String errorMessagePrefix = String.format("ContainerPlacement request: %s is rejected due to", requestMessage);
+    String errorMessagePrefix = ContainerPlacementMessage.StatusCode.BAD_REQUEST + " reason: ";
```

Comment:
Instead can we define errorMessage here 
as ContainerPlacementMessage.StatusCode.BAD_REQUEST + " reason: %s"

and later String.format(errorMessage, reason);